src/Application/Actions/Person/Update.php

```diff
@@ -161,6 +161,9 @@ protected function action(Request $request, array $args): Response
             );
         }
 
+        // We should persist Stripe's Customer ID on initial Person create.
+        \assert(is_string($person->stripe_customer_id));
```

Comment:
would it be worth adding something like `public function getStripeCustomerId(): string` to the person model, so we can re-use it in other places and to be a bit clearer?

The assertion does seem justified, I see we set the ID at https://github.com/thebiggive/identity/blob/af6e19e8f5341f4f301dab9cdead37d8140c25a9/src/Application/Actions/Person/Create.php#L141 and that seems to be the only place we create Persons. May be worth also just checking in the DB to make sure all old person records have the `stripe_customer_id` set.

Reply:
@bdsl There appear to be 2 historic Person records with no such ID, both from last year. All their non-timestamp, non-UUID fields are blank and I'm not aware of any current logic which would lead to this when things are working.

I think using a typed getter is fine.